### ISSUE TITLE
Multiple GeoJsonPopup leads to Uncaught SyntaxError: Identifier 'name_getter' has already been declared

### ISSUE THREAD

**Issue by orangeluc** [python-visualization/folium#1346]
Hi,

Have been trying to use make an interactive map with multiple data layers using folium. And for each layer, I would like to have tooltip and popups. Therefore I created functions to generate popups and tooltips.

However, there are issues to create multiple popups for multiple layers.

I think the issue could be link to the "name_getter" in the folium class GeoJsonPopup(GeoJsonDetail).

![image](https://user-images.githubusercontent.com/9092165/83753969-80276b80-a66b-11ea-917b-b4ec1f6a6448.png)

as I got this error 
![image](https://user-images.githubusercontent.com/9092165/83754076-a8af6580-a66b-11ea-8dbe-00be320a9c23.png)

For GeoJsonTooltip, it works pretty fine.

```python
from folium.features import GeoJson, GeoJsonTooltip, GeoJsonPopup
def create_popup(field):
    popup = GeoJsonPopup(
    fields=[field],
    aliases=[field],
    localize=True,
    labels=True,
    style="background-color: yellow;",
    ),
    parse_html=True
    return popup

def create_tooltip(field):
    tooltip = GeoJsonTooltip(fields=[field],
                             aliases=[field],
                             localize=True,
                             sticky=False,
                             labels=True,
                             style="""
                                 background-color: #F0EFEF;
                                 border: 2px solid black;
                                 border-radius: 3px;
                                 box-shadow: 3px;
                             """,
                             max_width=800,
                             parse_html=True

                             )
    return tooltip

inwoner = GeoJson(
    geo_data,
    style_function=lambda x: {
        "fillColor": colormap_inwoner(x["properties"]["INWONER"])
        if x["properties"]["INWONER"] is not None
        else "transparent",
        "color": "black",
        "fillOpacity": 0.4,
    },
    name = "Resident",
    tooltip=create_tooltip("INWONER"),
    popup=create_popup("INWONER")
).add_to(nl_map)


household = GeoJson(
    geo_data,
    style_function=lambda x: {
        "fillColor": colormap_hh(x["properties"]["AANTAL_HH"])
        if x["properties"]["AANTAL_HH"] is not None
        else "transparent",
        "color": "black",
        "fillOpacity": 0.4,
    },
    name = "Household",
    tooltip=create_tooltip("AANTAL_HH"),
    popup=create_popup("AANTAL_HH")
).add_to(nl_map)
```

**Comment by Conengmo** [comment]
Thanks for your report @orangeluc. I didn't test this but I can see how this error could occur. It seems that variable `name_getter` is not used anywhere so we could remove it. But we'd have to test to make sure. If someone wants to do that and open a PR that would be welcome.

**Comment by jtbaker** [comment]
I'll get the PR for this up, @Conengmo. I don't remember writing that variable assignment line, or understand why it is there since it seems to be not used, but it came from my PR for this feature it looks like.

**Comment by Conengmo** [comment]
@orangeluc with @jtbaker's help a fix got merged, so if you install folium from the master branch your error should be gone. The fix will also be in the next release, v0.12.0, date yet unknown.
